app/scripts/helpers/audioFactory.js

```diff
@@ -127,7 +131,11 @@ export function loadAudio(scene) {
  * @param {Phaser.Scene} scene
  */
 export function addSoundToScene(scene) {
-  AUDIO_COMMANDS.map((audio) => scene.sound.add(audio.command, audio.config) );
+  AUDIO_COMMANDS.forEach((audio) => {
+    Array.isArray(audio.command)
+      ? audio.command.forEach(cmd => scene.sound.add(cmd, audio.config))
+      : scene.sound.add(audio.command, audio.config)
+  });
```

Comment:
The `loadAudio` and `addSoundToScene` functions are very similar. It _might_ be worthwhile to abstract out the command to a higher order function, something like:
```js
function _helper(fn) {
  AUDIO_COMMANDS.forEach((audio) => {
    Array.isArray(audio.command)
      ? audio.command.forEach(cmd => fn(cmd, audio.config))
      : fn(audio.command, audio.config);
  )};
}

export function addSoundToScene(scene) {
  _helper(scene.sound.add);
}
```

At the same time it might be over optimization.